.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: Build
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  build:
+
+    env:
+      JAVA_OPTS: -Xms512m -Xmx1024m
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+        architecture: x64
+    - name: Install local dependencies
+      run: |
+        cd .. ; git clone https://github.com/arrow-kt/arrow-meta.git
+        cd arrow-meta
+        ./gradlew publishMeta
```

Comment:
I'll remove this temporary step as soon as `prelude` dependencies are unified (artifact id: `prelude` for local repository and `arrow-meta-prelude` for OSS repository now).